roles/BCPC-Hadoop-Head.json

```diff
@@ -15,10 +15,10 @@
     "recipe[bcpc::diamond]",
     "recipe[bcpc::graphite]",
     "recipe[bcpc::powerdns]",
+    "recipe[bach_krb5::krb5_client]",
     "recipe[bcpc-hadoop::disks]",
     "recipe[bcpc-hadoop::hdp_repo]",
     "recipe[bcpc-hadoop::configs]",
-    "recipe[pam::default]",
```

Comment:
If we remove `pam::default`, then we will be unable to set ulimits as `pam::default` is what actually writes out the Pam provider modified config files?